components/navbar/StatsDropdown.vue

```diff
@@ -1,72 +1,131 @@
-          {{ $t('stats') }}
-        </div>
+    <div v-if="!isMobile">
+      <b-dropdown
+        v-if="chain === 'bsx' || chain === 'snek'"
+        aria-role="list"
+        data-cy="stats"
+        :triggers="['click']">
+        <template #trigger>
+          <div class="navbar-item" data-cy="stats">
+            {{ $t('stats') }}
+          </div>
+        </template>
+        <b-dropdown-item has-link>
+          <nuxt-link
+            data-cy="global-offers"
+            :to="`${
+              accountId
+                ? `/${urlPrefix}/offers?target=${accountId}`
+                : `/${urlPrefix}/offers`
+            }`">
+            {{ $t('navbar.globalOffers') }}
+          </nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="offers-stats" :to="`/${urlPrefix}/stats`">
+            {{ $t('navbar.offerStats') }}
+          </nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="series-insight" to="/series-insight">
+            Series</nuxt-link
+          >
+        </b-dropdown-item>
+      </b-dropdown>
+      <b-dropdown v-if="chain === 'rmrk'" data-cy="stats" :triggers="['click']">
+        <template #trigger>
+          <div class="navbar-item" data-cy="stats">
+            {{ $t('stats') }}
+          </div>
+        </template>
+        <b-dropdown-item has-link aria-role="menu-item">
+          <nuxt-link data-cy="spotlight" to="/spotlight">
+            {{ $t('spotlight.page') }}
+          </nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="series-insight" to="/series-insight">
+            Series</nuxt-link
+          >
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="sales" to="/sales"> Sales</nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="hot" to="/hot"> Hot</nuxt-link>
+        </b-dropdown-item>
+      </b-dropdown>
+    </div>
+
+    <MobileExpandableSection
+      v-if="isMobile"
+      :no-padding="true"
+      :title="$t('stats')">
+      <template>
+        <template v-if="chain === 'bsx' || chain === 'snek'">
+          <b-navbar-item
+            data-cy="global-offers"
+            :to="`${
+              accountId
+                ? `/${urlPrefix}/offers?target=${accountId}`
+                : `/${urlPrefix}/offers`
+            }`"
+            tag="nuxt-link">
+            {{ $t('navbar.globalOffers') }}
+          </b-navbar-item>
+          <b-navbar-item
+            data-cy="offers-stats"
+            :to="`/${urlPrefix}/stats`"
+            tag="nuxt-link">
+            {{ $t('navbar.offerStats') }}
+          </b-navbar-item>
+          <b-navbar-item
+            data-cy="series-insight"
+            to="/series-insight"
+            tag="nuxt-link">
+            {{ $t('series.label') }}
```

Comment:
If we could use a map to manage these translation keys, that would be a plus. In this way, the same translation key could be used on pc and mobile.

like this
```
const translationKeyMap = {
    offerStats: this.$t('navbar.offerStats'),
    seriesInsight: this.$t('series.label'),
    ...
}

```

Reply:
For now, the translation words on mobile are all copied from that on the pc. They are totally the same. Anyway, this solution could be an optional plus.
> what if later on we want the texts to be different in both the places
Then we could remove the translation key from the map.